Assets/Scripts/Utilities/AudioZone.cs

```diff
@@ -0,0 +1,31 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class AudioZone : MonoBehaviour 
+{
+	[SerializeField] AudioSource sourceData = null;
+	AudioSource playingClip = null;
+
+	void OnTriggerEnter( Collider col )
+	{
+		if ( col.GetComponent<PlayerActor>() )
```

Comment:
I can this up with a bitfield that indicates which types this zone accepts and then then the zone tag includes an enum field. That way certain zones can accept only gods, only buddies, only the player, etc. Although maybe that enum field could just be on actor. Let me know when you're on chat so we can discuss this a bit.